myia/anf_ir.py

```diff
@@ -16,12 +16,15 @@
 
 from myia.ir import Node
 from myia.utils import Named, repr_, list_str
+from .primops import Return
```

Comment:
Missed this, but please don't do relative imports. (It's quite strongly discouraged.)

Reply:
Why do you say it's discouraged.  I've heard that about old-style relative imports (the kind that don't work in python 3), but never about the new style.

Reply:
> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path)

> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose.

Regardless, we should at least be consistent. So either all the imports should be relative, or all should be absolute, not a mix.

Reply:
What's discouraged are implicit relative imports like `from primops import ...`, not explicit relative imports like `from .primops import ...`. Your quote says these are an acceptable alternative. Personally I vote for relative imports.

Reply:
I just want to point out that non-explicit relative imports (as in `from primops import ...`) are no longer possible with python 3.  What @bartvm quoted is old advice that only applies to python 2.

In the current situation I slightly favor absolute import because we are bound to transform some files into modules and absolute imports will keep working if we do that.  In the long run I usually favour relative import because that's less typing.

Reply:
I'm going to change it for absolute import for consistency, we can figure out the final policy in another issue.

Reply:
@abergeron Advice I quoted is for the current PEP8 for Python 3. As @breuleux pointed out, guideline is that implicit relative imports are evil, explicit relative imports are "acceptable alternatives", absolute imports are "recommended".